cluster-provision/images/centos8-realtime/publish.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+docker tag quay.io/jordigilh/centos8-realtime-container-disk:latest quay.io/kubevirtci/centos8-realtime-container-disk:latest
+docker push quay.io/kubevirtci/centos8-realtime-container-disk:latest
```

Comment:
I've created the repository https://quay.io/repository/kubevirtci/centos8-realtime-container-disk and added you as maintainer.

Reply:
thanks!

Reply:
@dhiller changed and reused the `publish_image.sh` from project-infra since it has a lot of goodies that could become handy for any other image being generated here.
Let me know what you think :smile:

Reply:
You could also add `project-infra` as `extra_ref` as in this job: https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-postsubmits.yaml#L13
Docs here: https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md
This way you could use the same script and would avoid duplication.

Reply:
> I've created the repository https://quay.io/repository/kubevirtci/centos8-realtime-container-disk and added you as maintainer.

@dhiller , can you rename the repository to `https://quay.io/repository/kubevirtci/fedora-realtime-container-disk`? I ended up using Fedora and I got it working fine. Thanks!